03 Writing Algorithms/34 Algorithm Framework/02 Universe Selection/04 Fundamental Universes/02 Fundamental Selection.html

```diff
@@ -91,9 +99,12 @@
 # Outside of the algorithm class
 class LiquidAndLowPEUniverseSelectionModel(FundamentalUniverseSelectionModel):
     def select(self, fundamental: List[Fundamental]) -&gt; List[Symbol]:
+	# Select symbols with fundamental data and a price above $1.
         filtered = [x for x in fundamental if x.price &gt; 1 and not np.isnan(x.valuation_ratios.pe_ratio)]
-        most_liquid = sorted(filtered, key=lambda x: x.dollar_volume)[-100:]
-        lowest_pe_ratio = sorted(most_liquid, key=lambda x: x.valuation_ratios.pe_ratio)[:10]
+	# Sort descending by daily dollar volume.
+        most_liquid = sorted(filtered, key=lambda x: x.dollar_volume, reverse=True)[-100:]
```

Comment:
C# code sort by descending order. I changed the Python code to also sort in descending order for consistency.